pkg/api/common.go

```diff
@@ -12,11 +12,12 @@ import (
 	"io/ioutil"
 	"math"
 	"net/http"
-	"regexp"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/grafana/regexp"
```

Comment:
Why using this package instead of default golang one? I mean what are the benefits?

Reply:
Well, the point is to make outputs similar to what Prometheus does. Recently, Prometheus shifted from go regex to grafana regex. I don't know the reason, will try to see the godoc APIs. But, there might be an edge case in response comparison if we shifted back to the original.

Reply:
`This repo is a fork of the upstream Go regexp package, with some code optimisations to make it run faster.`

[Source](https://pkg.go.dev/github.com/grafana/regexp#section-readme)